assemblies/assembly-audit-log.adoc

```diff
@@ -34,6 +30,14 @@ Audit logs are not forwarded to the internal log store by default because this d
 [role="_additional-resources"]
 .Additional resources
 
-* For more information, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[{ocp-brand-name}  - About Logging]
+* For more information about logging in {ocp-short}, see link:https://docs.openshift.com/container-platform/latest/observability/logging/cluster-logging.html[About Logging]
+* For a complete list of fields that a {product-short} audit log can include, see xref:ref-audit-log-fields.adoc_{context}[]
+* For a list of scaffolder events that a {product-short} audit log can include, see xref:ref-audit-log-scaffolder-events.adoc_{context}[]
```

Comment:
[Docs] this renders as _see Section 9.2.1, “Audit log fields”_ and _see Section 9.2.2, “Scaffolder events”_  locally, but as IDs in the GitHub preview - is this expected behavior? Can we expect it to render correctly in Pantheon / on docs.redhat?